controllers/updates/views/listing.njk

```diff
@@ -0,0 +1,82 @@
+{% from "components/breadcrumb-trail/macro.njk" import breadcrumbTrail %}
+{% from "components/news.njk" import updateTeaser with context %}
+{% from "components/split-nav/macro.njk" import splitNav %}
+
+{% set bodyClass = 'has-static-header' %}
+{% extends "layouts/main.njk" %}
+
+{% block content %}
+    {% set pageAccent = 'cyan' %}
+
+    <main role="main" id="content">
+        <section class="content-box u-inner-wide-only accent--{{ pageAccent }} a--border-top">
+            {{ breadcrumbTrail(breadcrumbs) }}
+
+            <div class="{% if not isNewsLandingPage %}u-constrained-wide{% endif %}">
+                {% if entriesMeta.pageType === 'author' %}
+                    {% set activeAuthor = entriesMeta.activeAuthor %}
+                    <h1>Viewing all posts by {{ activeAuthor.title }}</h1>
+                {% elseif entriesMeta.pageType === 'tag' %}
+                    {% set activeTag = entriesMeta.activeTag %}
+                    <h1>Viewing all posts tagged {{ activeTag.title }}</h1>
+                {% elseif entriesMeta.pageType === 'category' %}
+                    {% set activeCategory = entriesMeta.activeCategory %}
+                    <h1>Viewing all posts in {{ activeCategory.title }}</h1>
+                {% else %}
+                    <h1>{{ title }}</h1>
+                {% endif %}
+
+                {% if isNewsLandingPage %}
+                    {% for type, entrylist in entries | groupItemsBy('updateType.name') %}
+                        {# @TODO find a better way to handle this #}
```

Comment:
Possibly we could do this replacement work at the Craft end so only the API needs to worry about the `_`/`-` replacement?

Reply:
Done (see the other PR!)